pkg/config/config.go

```diff
@@ -211,6 +212,11 @@ type PreallocSettings struct {
 	ForLoops   bool `mapstructure:"for-loops"`
 }
 
+type GodoxSettings struct {
+	IncludeTests bool `mapstructure:"include-tests"`
```

Comment:
looks like it's unused. And there is no need for such a setting because we can configure whether to run any specific linter on test files by `exclude-rules`

Reply:
Removed.